src/stars.css

```diff
@@ -0,0 +1,39 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
```

Comment:
why did you connect it twice? add it only in `style.css`

Reply:
Because when I add it only to style.css tests are failed (but in live server everything works).